drivers/vulkan/vulkan_context.cpp

```diff
@@ -234,6 +234,43 @@ Error VulkanContext::_create_validation_layers() {
 	return OK;
 }
 
+typedef VkResult (*_vkEnumerateInstanceVersion)(uint32_t *);
+
+void VulkanContext::_obtain_vulkan_version() {
+	// init to Vulkan 1.0
+	vulkan_major = 1;
+	vulkan_minor = 0;
+	uint32_t vulkan_patch = 0; // we don't track this one because we might override our major/minor
+
+	// https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkApplicationInfo.html#_description
+	// for Vulkan 1.0 vkEnumerateInstanceVersion is not available, including not in the loader we compile against.
+	_vkEnumerateInstanceVersion func = (_vkEnumerateInstanceVersion)vkGetInstanceProcAddr(NULL, "vkEnumerateInstanceVersion");
+	if (func != NULL) {
+		uint32_t api_version;
+		// VkResult res = vkEnumerateInstanceVersion(&api_version);
```

Comment:
I guess this can be removed, it's clear from the code just above that it's what `func` resolves to.
If it's for documentation purpose, we could instead make the `func` name more explicit, e.g. `vk_version_func` (but again as the `vkGetInstanceProcAddr` call is two lines above, I guess it's fine).

Reply:
yeah that was more left because the function call does exist in the loader BUT not in the loader version in Android. I might add a remark to that effect so people know why we're getting the function pointer instead of just using the function in the loader.